plugins/BEdita/API/src/Controller/LoginController.php

```diff
@@ -65,7 +65,7 @@ public function initialize()
                             'Weak' => ['hashType' => 'md5'],
                         ],
                     ],
-                    'finder' => 'login',
+                    'finder' => 'userLogin',
```

Comment:
Uhm… `login` finder should be enough here. `userLogin` finder should be redundant.

Reply:
And what about changing scope config like:
```php
    'scope' => [
        'blocked' => false,
        'deleted' => false,
        'status IN' => ['on', 'draft']
    ],
```
without custom finder?

Reply:
Should be fine, but looking forward to migrating to Auth plugin, `scope` will not be available any more, so if using a finder doesn't cause too much hassle, that should be the preferred way to go.

Reply:
ok, let's use a finder then

Reply:
done